rest/route/degraded_mode.go

```diff
@@ -0,0 +1,51 @@
+package route
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/evergreen-ci/evergreen"
+	"github.com/evergreen-ci/gimlet"
+	"github.com/pkg/errors"
+)
+
+const firingStatus = "firing"
```

Comment:
Since I don't know what the alertmanager setup looks like, is this the alert specifically for CPU, or are there other alerts that could trigger this? Making the check as specific as possible would be nice to make sure we don't degrade for random alerts.